pages/index.vue

```diff
@@ -59,5 +59,11 @@ import VuetifyLogo from '~/components/VuetifyLogo.vue'
     VuetifyLogo
   }
 })
-export default class IndexPage extends Vue {}
+export default class IndexPage extends Vue {
+  head(this: IndexPage): object {
+    return {
+      title: 'Home'
```

Comment:
Do we want a TODO here if we want a different html title for the home page? like the app name?

Reply:
I don't think so. This part of the code will get change by the developer once they start developing. Also, this doesn't override the entire page title, just the `%s` part.